tools/utility_functions.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Bash utility functions used within the project.
+
+function stage() {
+    BOLD_BLUE="\e[1m\e[96m"
```

Comment:
As I've reported in other review, the ancient version of Bash (3.2) distributed with macOS does not support control sequences in `echo -e`. This turns the eyecandy into an eyesore. One way to fix it is to create a function for printing eyecandy, like this:

```shell
function eye_candy() {
   if [[ ${BASH_VERSINFO[0]} < 4 ]]; then
        color=""
    else
        BOLD_BLUE="\e[1m\e[96m"
        RED="\e[31m"
        YELLOW="\e[33m"
        RESET="\e[0m"
        color=$(eval echo \$$1)
    fi
    if [ -n "${color}" ]; then
        echo -e "${color}${2}${RESET}"
    else
        echo $2
    fi
}
```

and then use it wherever eye candy is desired, like this:

```shell
function stage() {
    eye_candy BOLD_BLUE $1
}
```

Reply:
Done